terrawrap/utils/config.py

```diff
@@ -22,6 +22,8 @@
 
 DEFAULT_REGION = 'us-west-2'
 SSM_ENVVAR_CACHE = SSMParameterGroup(max_age=600)
+TF_WRAP_FILE = ".tf_wrapper"
+TF_WRAP_FILE_YML = ".tf_wrapper.yml"
```

Comment:
I don't think we need to support this filename. I don't see any cases where we use `.tf_wrapper.yml` in our code

Reply:
Done